vllm/model_executor/layers/quantization/utils/w8a8_utils.py

```diff
@@ -7,8 +7,7 @@
 
 # Input scaling factors are no longer optional in _scaled_mm starting
 # from pytorch 2.5. Allocating a dummy tensor to pass as input_scale
-TORCH_DEVICE_IDENTITY = torch.ones(1).cuda() \
-            if current_platform.is_rocm() else None
```

Comment:
Do you know why it was checking for is_rocm here?

Reply:
The assumption being made was `is_rocm == torch 2.5`. If we really wanted to we could check the torch version, but I think we can just assume torch 2.5 as a floor now